dmn/src/test/java/io/camunda/zeebe/dmn/DmnEvaluationTest.java

```diff
@@ -50,7 +50,7 @@ void shouldResultInFailureWhenDecisionNotInDrg() {
         .isNotNull()
         .describedAs(
             "Expect that the evaluation failed because the DRG does not contain the referred decision")
-        .contains("no decision found for 'not_in_drg'");
+        .contains("no decision found with id 'not_in_drg'");
 
```

Comment:
I'm assuming this change is due to:
- https://github.com/camunda/dmn-scala/pull/206

❓ Do we need to make other adjustments in Zeebe to use this new behavior, or to improve the error messages in Zeebe for this case?

Reply:
Good hint. :+1: 

I removed the special handling for this bug from the DMN integration. 

Otherwise, we don't need any adjustments.